fms/testing/_internal/model_test_suite.py

```diff
@@ -171,9 +172,13 @@ def test_model_output(self, model, signature, capture_expectation):
                 "Signature file has been saved, please re-run the tests without --capture_expectation"
             )
 
-        assert np.allclose(
-            np.array(actual), np.array(signature)
-        ), _FAILED_MODEL_SIGNATURE_OUTPUT_MSG
+        assertion_msg = f"""
+        difference: {np.mean(np.abs(np.array(actual) - np.array(signature)))}
+        
+        {_FAILED_MODEL_SIGNATURE_OUTPUT_MSG}
+        """
+
+        assert np.allclose(np.array(actual), np.array(signature)), assertion_msg
```

Comment:
as in the other PR, you might want to use `torch.testing.assert_close` instead

Reply:
in this case we have a list, not a tensor. Would this be ok then?